README.md

```diff
@@ -92,6 +92,52 @@ gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys <last eight symbols of
 gpg --keyserver hkp://keyserver.ubuntu.com --send-keys <last eight symbols of gnupg keyId>
 ```
 
+### Stencil Server API
+
+For serving the protobuf descriptor set artifacts and their versions we use a Stencil Server.
```

Comment:
Can avoid terms like `We` and make it generic to users.

Reply:
Can you please suggest the changes, or point me to a written guideline?

Reply:
There are no written guidelines. This codebase is a generic library and you're augmenting it with a utility more of how it could managed behind an http server, it's a recommendation and a pattern.